pyodide_env.sh

```diff
@@ -17,3 +17,13 @@ export EM_DIR
 export _EMCC_CCACHE=1
 # mtime of this file is checked by ccache, we set it to avoid cache misses.
 export EM_CONFIG="$ROOT/emsdk/emsdk/.emscripten"
+
+# Export SED variable if not already set
+# Use GNU sed (gsed) if available, otherwise fallback to sed
+if [ -z "$SED" ]; then
+    if command -v gsed >/dev/null 2>&1; then
+        export SED="gsed"
+    else
+        export SED="sed"
+    fi
+fi
```

Comment:
This is not needed if we don't use `SED` alias in recipes.

Reply:
On my machine, the build works with the entire patch, but not without this. So patching only Makefile will not make much enhancement.
I'd better to find another way to fully inject gsed to the build step